libvast/include/vast/concept/parseable/vast/option_set.hpp

```diff
@@ -0,0 +1,104 @@
+//    _   _____   __________
+//   | | / / _ | / __/_  __/     Visibility
+//   | |/ / __ |_\ \  / /          Across
+//   |___/_/ |_/___/ /_/       Space and Time
+//
+// SPDX-FileCopyrightText: (c) 2023 The VAST Contributors
+// SPDX-License-Identifier: BSD-3-Clause
+
+#pragma once
+
+#include "vast/concept/parseable/core/parser.hpp"
+#include "vast/concept/parseable/string/char.hpp"
+#include "vast/concept/parseable/string/char_class.hpp"
+#include "vast/concept/parseable/vast/data.hpp"
+#include "vast/data.hpp"
+
+#include <algorithm>
+
+namespace vast {
+using namespace parser_literals;
```

Comment:
This should not be at the namespace level here, as that adds too much magic just for including this file.

Reply:
I think it is not needed at all in the current implementation. I tried compiling the previous version without it and it worked

Reply:
Removed in https://github.com/tenzir/vast/pull/2877/commits/c56df12445c4ab032cd0aca5d6d3ab8d993b619d